developers/pymavlink/change_flight_mode.py

```diff
@@ -38,8 +38,10 @@
     ack_msg = master.recv_match(type='COMMAND_ACK', blocking=True)
     ack_msg = ack_msg.to_dict()
 
-    # Check if command in the same in `set_mode`
-    if ack_msg['command'] != mavutil.mavlink.MAVLINK_MSG_ID_SET_MODE:
+    # Continue waiting if the acknowledged command is not `set_mode`
+    if ack_msg['command'] != mavutil.mavlink.MAV_CMD_DO_SET_MODE:
+        # Would be good to add mechanism to avoid endlessly blocking
+        # if the autopilot sends a NACK or never receives the message
```

Comment:
This comment should be up at line 38 where the blocking receive actually occurs.